src/Compilers/Core/Portable/Operations/ICollectionElementInitializerOperation.cs

```diff
@@ -1,23 +1,23 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
+using System;
 using System.Collections.Immutable;
 
 namespace Microsoft.CodeAnalysis.Operations
 {
     /// <summary>
-    /// Represents a collection element initializer.
+    /// Obsolete interface that used to represent a collection element initializer. It has been replaced by
+    /// <see cref="IInvocationOperation"/> and <see cref="IDynamicInvocationOperation"/>, as appropriate.
     /// <para>
     /// Current usage:
-    ///  (1) C# nested collection element initializer expression within a collection initializer.
-    ///      For example, given a collection initializer "new Class() { Y = { { x, y, 3 } } }",
-    ///      nested collection element initializer for Y, i.e. "{ { x, y, 3 } }" is represented by this operation.
-    ///  (2) VB nested collection element initializer expression within a collection initializer.
+    ///   None. This API has been obsoleted in favor of <see cref="IInvocationOperation"/> and <see cref="IDynamicInvocationOperation"/>.
     /// </para>
     /// </summary>
     /// <remarks>
     /// This interface is reserved for implementation by its associated APIs. We reserve the right to
     /// change it in the future.
     /// </remarks>
+    [Obsolete]
```

Comment:
>[Obsolete] [](start = 4, length = 10)

I am not sure what is our policy about marking existing APIs as obsolete. Might want to check with Compatibility Council

Reply:
Still figuring this out, has not yet been addressed.

---
In reply to: [189329437](https://github.com/dotnet/roslyn/pull/26781#discussion_r189329437) [](ancestors = 189329437)